Unity Example/Assets/Discord RPC/Scripts/Presence/DiscordPresence.cs

```diff
@@ -151,6 +182,15 @@ public DiscordRPC.RichPresence ToRichPresence()
 			if (endTime.IsValid()) presence.Timestamps.End = endTime.GetDateTime();
 		}
 
+		if (buttons.Length > 0)
+		{
+			presence.Buttons = new DiscordRPC.Button[]
+			{
+				new DiscordRPC.Button { Label = buttons[0].label, Url = buttons[0].url},
+				new DiscordRPC.Button { Label = buttons[1].label, Url = buttons[1].url}
+			};
```

Comment:
This shouldn't be a new array of fixed length. Buttons can be any length. See other review.

Reply:
fixed